frontend/src/app/helpers/spec-helper/element.spec-helper.ts

```diff
@@ -0,0 +1,69 @@
+/*
```

Comment:
What do you think about moving this file to the `tests` directory? It is test specific and I'd expect helpers in the `src` directory that are also used in the "production" code (except the `.spec.ts` files).

Reply:
The problem at the moment is that the `tests` directory is not inside the `/src` directory, which makes sense to me as this way we have a clear separation between the unit/integration tests and the E2E tests. So I personally would prefer to keep it that way. However, we could create another folder inside the `/src` directory where we then put the spec helpers.

Reply:
In theory, they don't need to be part of the `tests` folder. You can still reference them relatively.